packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -1043,7 +1043,9 @@ addFieldsDict(Users, {
       else if (isReviewed && karma>=20) return 1.0;
       else if (isReviewed && karma>=0) return 0.9;
       else if (isReviewed) return 0.8;
-      else if (signUpReCaptchaRating>=0) {
+      else if (signUpReCaptchaRating !== null && 
+              signUpReCaptchaRating !== undefined && 
+              signUpReCaptchaRating>=0) {
```

Comment:
`null >= 0` is true in JavaScript, so a signUpReCaptchaRating of null passed this check. Worse, `null * 0.8` is 0, so this returns the worst possible spam risk score.